fcb-source/utilities/src/flash.c

```diff
@@ -214,7 +244,7 @@ static FlashErrorStatus ReadFlashPage(uint8_t * readData, const uint8_t pageNbr)
  * @retval word value as an unsigned 32-bit integer
  */
 static uint32_t ReadFlashWord(const uint8_t pageNbr, const uint16_t wordNbr) {
```

Comment:
Now the variable wordNbr will not actually be the word number, but rather the byte number?